plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -1043,23 +1079,26 @@ var _ = Describe("validator", func() {
 		})
 
 		Context("tests for unknown provider", func() {
-			Context("scheduling checks", func() {
+			Context("scheduling checks: w/o binding subresource", func() {
```

Comment:
I think that it is a bit clearer if you specify that these tests are for the `Create` operation.
```suggestion
			Context("scheduling checks for Create operation", func() {
```